src/_pytest/pathlib.py

```diff
@@ -365,3 +365,105 @@ def fnmatch_ex(pattern: str, path) -> bool:
 def parts(s: str) -> Set[str]:
     parts = s.split(sep)
     return {sep.join(parts[: i + 1]) or sep for i in range(len(parts))}
+
+
+def import_module(path, modname=None, ensuresyspath=True):
```

Comment:
Since `_pytest.pathlib` usually deals with `pathlib.Path`s, would be good to annotate `path: py.path.local` here to avoid confusion.

Reply:
lets annotate a 'pathLike or Path, and cast to Path if possible?